templates/components/form/form.jsx

```diff
@@ -78,12 +80,11 @@ var Form = React.createClass({
   },
   render: function() {
      var fields = Object.keys(this.props.fields).map(this.buildFormElement);
-     return <div role="form">{fields}</div>;
+     return <div role="form"><form>{fields}</form></div>;
```

Comment:
we already have div[role=form]. Turning it into a magically active HTML element is probably a really bad plan